src/PassengerClass.js

```diff
@@ -0,0 +1,16 @@
+const { bookshelf, Passenger, Passengers, knex } = require('../database/connection.js')
+const commands = require('./commands.js')
+
+class PassengerClass {
```

Comment:
@jrob8577 Yeah, I was unhappy with that name.  
The Model in the connection.js file is also named Passenger, after the table.  I wanted to avoid confusion in files such as commands.js where we might need to refer to both this class and the model.  What would be a better way of handling this?